templates/centos6.tpl.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
-  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}-v0.7.0
+  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}-v{{ version }}
```

Comment:
I think much better solution would be to set the version variable here (see line 14 for an example of how to set template variable). That will keep the version tied to the proper template file. And the include _linux.yaml will then propagete it to the common template.

Also, you should probably add a comment to the template explaining that the version must never change as it is part of the template identity. A change to version = change to VM compatibility.

Reply:
That's a good idea, will do that, thanks!